### DISCUSSION TITLE
Welcome to My-First-Repo Discussions!

### DISCUSSION THREAD

**Discussion by DanMathew7** [DanMathew7/My-First-Repo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlanBinu007** [comment]
Welcome to GitHub Bro!!
